src/js/utils/index.ts

```diff
@@ -0,0 +1,29 @@
+import { Utils } from "@mat3ra/utils";
+
+import { TestCaseHandler } from "./TestCaseHandler";
+
+export function generateTestFeaturesFromYAMLConfig(yamlContent: string, templateContent: any) {
```

Comment:
Should be generateTestFeaturesFromYAMLConfig and take templateContent, testCases, testCaseSchema